docs/administrator-docs/installing.md

```diff
@@ -20,11 +20,23 @@ The Jellyfin Docker image is available on [Docker Hub](https://hub.docker.com/r/
     `--name jellyfin \`  
     `--volume /mnt/jellyfin_config:/config \`  
     `--volume /mnt/jellyfin_cache:/cache \`  
-    `--volume /path/to/media:/mnt/library \`  
-    `--publish 8096:8096 \`  
+    `--volume /path/to/media:/media:ro \`  
     `--net=host \`  
-    `jellyfin/jellyfin:latest`
-
+    `jellyfin/jellyfin:latest`  
+  
+Alternative docker-compose example:  
+```
+    version: "3"  
+    services:  
+        jellyfin:  
+          container_name: jellyfin  
```

Comment:
This is unnecessary for Docker Compose.

Ex. You can still do this without `container_name`.

```
docker-compose exec jellyfin ls
```

Reply:
I agree that this isn't necessary, but the resulting container will have a name of `<stack_name>_jellyfin_1` that I don't feel is as intuitive.

Also, the original example used the `--name` parameter, so specifying the `container_name` keeps the two methods consistent with one another.

Reply:
There's little reason to keep them consistent since the docker-compose command negates the need to have the container name be friendly. Please remove.